src/app/pages/display/display.component.html

```diff
@@ -198,3 +198,20 @@
     <mat-spinner></mat-spinner>
   </div>
 </ng-template>
+
+<ng-template #countdown >
+<section class="countdown p-4 shadow-2">
+  <div>
+    <mat-progress-spinner
+        class="progressBar"
```

Comment:
```suggestion
        class="progress-bar"
```
I'll probably make the css follow BEM, but it's good to get used to using kebab-case for the css classes.